CONTRIBUTING.md

```diff
@@ -1,100 +1,93 @@
-get confirmation from the reviewers that their planned mechanism for providing
-access to the implementation of the feature is acceptable.
-
-It is expected that during this implementation phase there will be changes
-made to the design and proposed specification edits to accurately represent
-the current status of the proposal.
-
-If the proposal adds or changes an object model or resource endpoint definition,
-it is expected that the PR will also include the necessary updates to the
-[Open API document](openapi.yaml).
-
-Once support for the change has been implemented, a PR should be sent to this
-repo for the change to the Open Service Broker API specification. By this
-point, the API interactions should be well understood and there should be no
-technical surprises; we expect the only discussion necessary on PRs to be for
-wordsmithing and formatting.
-
-## GitHub Labels
-
-When new [design proposals](#design-change-proposals) for the API specification
-are submitted, the working group will use a set of pre-defined GitHub labels to
-highlight the current stage of the proposal. Note that these labels will not be
-used for [minor changes](#minor-changes).
-
-The labels that will be used are as follows:
-- `1 - reviewing proposal`: The API working group is actively reviewing a
-proposal that has been submitted as a GitHub Issue with the aim of validating
-both the problem statements and any proposed solutions.
-- `2 - validating through implementation`: One of more platforms are actively
-working on the proposal with the aim of providing feedback on the implementation
-of the proposed solution.
-- `3 - reviewing PR`: Feedback has been received on the implementation of the
-proposed solution and a Pull Request has been created containing the validated
-specification changes.
-
-Note that not all issues will require a validation through implementation stage,
-and proposals can move back to a previous label at any time. When a PR is
-merged, the associated issue should be closed and any labels removed from the
-issue.
+as on our weekly calls. Once the working group have agreed to focus on a
+proposal, it will move into the first stage, as described below.
+
+The stages that a proposal will go through are as follows:
+- `1 - reviewing proposal`:
+  The API working group is actively reviewing a proposal that has been submitted
+  as a Github issue with the aim of validating both the problem statements and
+  any proposed solutions. Critical information such as new API endpoints and
+  response codes should be reviewed at this stage. For changes largely impacting
+  the Open Service Broker API actors (Platforms, Service Authors, etc), it is
+  recommended to solicit feedback from these actors and leave enough time (say
+  2 weeks) for feedback to be provided, and for the potentially received
+  objections/suggestions to be handled.
+  Once the design has been finalized in the Google Doc, the proposed set of
```

Comment:
One thing I think we should note here that comes from the F2F - it might be helpful to mention that small, focused changes are easier to make progress on.